Rubberduck.Inspections/Abstract/InspectionResultBase.cs

```diff
@@ -60,13 +60,14 @@ public string ToClipboardString()
             var documentName = Target != null 
                 ? Target.ProjectDisplayName 
                 : string.Empty;
-            if (string.IsNullOrEmpty(documentName))
-            {
-                var component = module.Component;
-                documentName = component != null 
-                    ? component.ParentProject.ProjectDisplayName 
-                    : string.Empty;
-            }
+            //todo: Find a sane way to reimplement this.
```

Comment:
outstanding todo

Reply:
I know. I just have no idea how to reimplemet this in a good way.  (Inspections results get newed up.)
Moreover, unless some XALM binding is using this, nothing really uses this method.